### DISCUSSION TITLE
How dotfiles "Update" feature works?

### DISCUSSION THREAD

**Discussion by Bammede** [mylinuxforwork/dotfiles#54] [Q&A]
if I change some settings, files in dotfiles will they be lost after update? How to preserve them?

**Comment by mylinuxforwork** [comment]
Hi. I have pushed today an update to support exactly in this use case. 

https://github.com/mylinuxforwork/dotfiles/wiki/Hook-and-Post-Installation-Scripts

But you need to run the installer once. So please make a backup with the installer and restore your changes one time.

Then follow the wiki article to protect your changes.
